cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering your models with the Django admin site to manage them through the admin interface. You can do this by importing your models and using `admin.site.register(YourModel)`.